salt/modules/archive.py

```diff
@@ -936,7 +936,7 @@ def cmd_unzip(zip_file,
     if password:
         cmd.extend(['-P', password])
     if options:
-        cmd.append('{0}'.format(options))
+        cmd = cmd.extend(shlex.split(options))
```

Comment:
Just change this line to:
```
cmd.extend(shlex.split(options))
```
extend is an inline modifier so you are setting `cmd` to None here